packages/client/src/lib.js

```diff
@@ -215,6 +215,20 @@ class Web3Storage {
     return res.json()
   }
 
+  /**
+   * @param {Service} service
+   * @param {{before: string, size: number}} opts
+   * @returns {Promise<Response>}
+   */
+  static async list ({ endpoint, token }, { before = new Date().toISOString(), size = 25 }) {
+    const search = new URLSearchParams({ before, size: size.toString() })
```

Comment:
Nit: I think unnecessary to to to toString.

```suggestion
    const search = new URLSearchParams({ before, size })
```

<img width="331" alt="Screenshot 2021-07-27 at 15 45 35" src="https://user-images.githubusercontent.com/152863/127175149-33504307-67f8-442d-9d6d-22164d9eca31.png">

Reply:
oh yes it is very uneccessary. but guess what. `tsc` hates it.

Reply:
ohhhh